src/components/PaginationFooter/Dropdowns/DropDown.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import type { SelectChangeEvent } from '@mui/material/Select';
+import { usePaginationContext } from '../context/PaginationProvider';
+import { StyledSelect, StyledTypography } from '../styles';
+
+interface Props {
+  name: string;
+  value: number;
+  options: JSX.Element;
+  handleChange: (event: SelectChangeEvent<number>) => void;
+}
+
+const DropDown = ({ name, value, options, handleChange }: Props) => {
+  const { layout, footerStyle, translator, tabIndex } = usePaginationContext();
+  const chartId = layout.qInfo.qId as string;
+  const label = translator.get(`NebulaTableUtils.Pagination.${name}`);
+  const baseId = `${name}-dropdown`;
+  const labelId = `${baseId}-label-${chartId}`;
```

Comment:
is this intended to be like below? or am i wrong in this case? (at least it's name `labelId` pointed me into this direction)
```suggestion
  const labelId = `${baseId}-${label}-${chartId}`;
```

Reply:
The current code is correct. `label` is just the translated string. I guess the "label" string is there so that it won't conflict with other components. But it's just a copy paste from sn-table.